lib/src/component/prop_mixins.dart

```diff
@@ -72,19 +72,26 @@ abstract class _$DomPropsMixin {
 
   int cols, rows, size, span, start;
 
-  bool allowFullScreen, async, autoPlay, checked, controls, defer, disabled, formNoValidate, hidden, loop, multiple,
-    muted, noValidate, open, readOnly, required, seamless, selected;
+  @Accessor(key: 'default')
+  bool htmlDefault;
+
+  @Accessor(key: 'is')
+  String htmlIs;
+
+  bool allowFullScreen, async, autoPlay, checked, controls, defer, disabled, formNoValidate, headers, hidden,
+      inputMode, integrity, keyParams, keyType, kind, loop, multiple, muted, noValidate, open, readOnly, required, reversed,
+      scoped, seamless, selected;
 
   Map<String, dynamic> style;
 
-  String className, title, id;
+  String challenge, cite, className, controlsList, formAction, formEncType, formMethod, formTarget, nonce, summary, title, id, wrap;
 
-  dynamic accept, acceptCharset, accessKey, action, allowTransparency, alt, autoComplete, cellPadding, cellSpacing,
-    charSet, classID, colSpan, content, contentEditable, contextMenu, coords, crossOrigin, data, dateTime,
-    dir, download, draggable, encType, form, frameBorder, height, href, hrefLang, htmlFor, httpEquiv, icon, label,
-    lang, list, manifest, max, maxLength, media, mediaGroup, method, min, name, pattern, placeholder,
-    poster, preload, radioGroup, rel, role, rowSpan, sandbox, scope, scrolling, shape, sizes, spellCheck, src, srcDoc,
-    srcSet, step, tabIndex, target, type, useMap, value, width, wmode;
+  dynamic accept, acceptCharset, accessKey, action, allowTransparency, alt, autoComplete, capture, cellPadding, cellSpacing,
```

Comment:
In this section there's name, pattern, placeholder, src, srcdoc, and srcLang that are marked as `String` by TS / React but listed here as `dynamic`. Was there a reason behind that?

Reply:
For both your questions, the fields you mentioned were already existing. I updated the already existing SvgProps typing, but I never did the same for the existing DomProps. I'll update those. Thank you for checking these!

Reply:
Also, when I updated the typing for the existing SvgProps, I made the PR into master as well (#356). Should I do the same for these updates?

Reply:
PR #375 updates to these existing DomProps types in master as well.